agent/build.gradle

```diff
@@ -73,16 +82,29 @@ shadowJar {
     exclude 'META-INF/services/javax.servlet.ServletContainerInitializer'
     exclude 'mozilla/**'
     exclude 'commons-logging.properties'
+}
+
+task shadowJar2(type: com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar) {
+
+    dependsOn shadowJar
+
+    from {
+        zipTree(shadowJar.archiveFile)
+    }
+
+    configurations = [project.configurations.runtime]
+
+    archiveClassifier = ''
+    mergeServiceFiles()
+
+    dependencies {
+        include(dependency('org.glowroot.xyzzy.instrumentation:'))
+    }
 
     transform(com.github.jengelman.gradle.plugins.shadow.transformers.AppendingTransformer) {
         resource = 'META-INF/xyzzy.instrumentation.list'
     }
-}
-archivesBaseName = 'applicationinsights-agent'
 
-jar {
-    dependsOn shadowJar
-    enabled = false
     manifest {
```

Comment:
Why did the manifest config move from the jar task?

Reply:
now there are 2 tasks that extend jar

Reply:
makes sense